.gitignore

```diff
@@ -3,3 +3,4 @@ coverage/
 .twilio-functions
 package-lock.json
 .DS_Store
+.env
```

Comment:
Can you broaden this pattern to match for the subdirectory (apps) .env files? As is this will only impact the root directory which does not have much risk of an accidental .env file being committed moving forward

Reply:
I broadened this to `**/.env`.